src/generators/schema/howto.php

```diff
@@ -0,0 +1,192 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package Yoast\WP\Free\Presentations\Generators\Schema
+ */
+
+namespace Yoast\WP\Free\Presentations\Generators\Schema;
+
+use Yoast\WP\Free\Context\Meta_Tags_Context;
+use Yoast\WP\Free\Helpers\Schema\Image_Helper;
+
+/**
+ * Returns schema FAQ data.
+ *
+ * @since 11.5
+ */
+class HowTo extends Abstract_Schema_Piece {
+
+	/**
+	 * Holds the allowed HTML tags for the jsonText.
+	 *
+	 * @var string
+	 */
+	private $allowed_json_text_tags = '<h1><h2><h3><h4><h5><h6><br><ol><ul><li><a><p><b><strong><i><em>';
+
+	/**
+	 * Determines whether or not a piece should be added to the graph.
+	 *
+	 * @param Meta_Tags_Context $context The meta tags context.
+	 *
+	 * @return bool
+	 */
+	public function is_needed( Meta_Tags_Context $context ) {
+		return ! empty( $context->blocks[ 'yoast/how-to-block' ] );
+	}
+
+	/**
+	 * Renders a list of questions, referencing them by ID.
+	 *
+	 * @param Meta_Tags_Context $context The meta tags context.
+	 *
+	 * @return array $data Our Schema graph.
+	 */
+	public function generate( Meta_Tags_Context $context ) {
+		$graph = [];
+
+		foreach ( $context->blocks[ 'yoast/how-to-block' ] as $index => $block ) {
+			$data = [
+				'@type'            => 'HowTo',
+				'@id'              => $context->canonical . '#howto-' . $index,
+				'name'             => $context->title,
+				'mainEntityOfPage' => array( '@id' => $context->main_schema_id ),
+				'description'      => '',
+			];
+
+			$json_description = \strip_tags( $block['attrs']['jsonDescription'], '<h1><h2><h3><h4><h5><h6><br><ol><ul><li><a><p><b><strong><i><em>' );
```

Comment:
The second argument is exactly the value of the `$allowed_json_text_tags` property.